model_templates/training/python3_sklearn/push.yaml

```diff
@@ -0,0 +1,8 @@
+model:
+    name: drumpush-regression
+    type: training
+    targetType: regression
+    environmentID: 5e8c889607389fe0f466c72d
```

Comment:
Using an ID is not really user facing.. do we have other options? Like using a name?

Reply:
We can use a name but then people have to spell it right which is risky

Reply:
We could probably do some interesting regex to accomplish this though

Reply:
Often in such cases slugs are used as user-facing identifiers.
This repo already has unique slugs internally ("python3_keras", "python3_sklearn"), which could be easy to refer to.
And then if you look, push.yaml already lives under "python3_sklearn" env identifier, so its more or less aware.
Downside: I havent seen similar patterns in DR except maybe layout/overlay names in DRCA, + maintaining constraints within DR.

Reply:
(Remembered that we slugify organization names but use them afaik internally)